### ISSUE TITLE
Bump build_config from ^0.2.5 to ^0.3.0

### ISSUE THREAD

**Issue by asaarnak** [pulyaevskiy/node-interop#28]
First thanks for this great project!

Seems that build_node_compilers doesn't play well with newest build_runner.

Using functions interop with:
```yaml
  build_runner: ^0.8.10
  build_node_compilers: ^0.1.4
```
Gives error:
```bash
Because build_node_compilers >=0.1.2 depends on build_config ^0.2.5 and build_runner >=0.8.10 depends on build_config ^0.3.0, build_node_compilers >=0.1.2 is incompatible with build_runner >=0.8.10.
So, because functions depends on both build_runner ^0.8.10 and build_node_compilers ^0.1.4, version solving failed.
pub get failed (1)
```

**Comment by pulyaevskiy** [comment]
Ouch, this is not good.

Thanks for reporting this! I should have some time later today to look into this.

**Comment by pulyaevskiy** [comment]
Published as `0.1.5`.
